app/models/custom_label.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class CustomLabel
+  def initialize(attrs)
+    @attrs = attrs
+  end
+
+  def id
+    @attrs['id']
+  end
+
+  def display_label
+    @attrs['displayLabel']
+  end
+
+  def display_on_full_text_finder
+    @attrs['displayOnFullTextFinder']
+  end
+
+  def display_on_publication_finder
+    @attrs['displayOnPublicationFinder']
+  end
+
+  def self.all
+    rmapi_customer_root.all.labels.map { |attrs| new attrs }
+  end
+
+  def self.update(label_id, attrs)
+    id = label_id.to_i
+    return unless id.positive?
```

Comment:
We shouldn't have different return values from a single method. In other words, we should still return a custom label no matter what, not `nil` in one case, and `new attrs` in another.

Reply:
This was more like a defense mechanism, in case the consumer provides some invalid value for the update like -1. was thinking more on lines of logging and reporting an error that the provided value is invalid. Are you suggesting we just return an empty response without any custom labels instead?

Reply:
Ok, realized that validations could be done in the controller.. will update code accordingly